.circleci/workflows/openlineage-java.yml

```diff
@@ -9,11 +9,11 @@ workflows:
       - build-integration-spark:
           matrix:
             parameters:
-              spark-version: [ '2.4.6', '3.1.2', '3.2.1' ]
```

Comment:
Looks like a wrong git conflict resolution.
Any particular reason for that?

Reply:
Yes, my mistake in manual resolution :)

Fixed.